pkg/gui/keybindings.go

```diff
@@ -268,6 +268,13 @@ func (gui *Gui) GetInitialKeybindings() []*Binding {
 			Handler:     gui.handleContainersExecShell,
 			Description: gui.Tr.ExecShell,
 		},
+		{
+			ViewName:    "services",
+			Key:         'E',
+			Modifier:    gocui.ModNone,
+			Handler:     gui.handleContainersExecShell,
```

Comment:
I believe this will shell into the selected container in the containers panel, but we want it to shell into the container of the selected service in the services panel. Could you create a new function for that in `services_panel.go`?

Reply:
Actually, I just discovered that `E` works on the services panel too :confused: . I don't know how this happened, perhaps my package manager had an older version and then it was fixed. So I think this PR doesn't make any sense anymore.